docs/src/main/sphinx/installation/cli.rst

```diff
@@ -144,6 +144,8 @@ mode:
     - Configures the level of detail provided for network logging of the CLI.
       Defaults to ``NONE``, other options are ``BASIC``, ``HEADERS``, or
       ``BODY``.
+  * - ``--no-progress``
```

Comment:
`progress` isn't documented already so it needs to be there and it should include mentioning that it can be negated instead of separate entry altogether maybe.

Reply:
The options here are slightly modified and broken down into parts, it's not just a copy of the output of `--help`.

Reply:
Agree with your setup and comments @nineinchnick

Reply:
`--no-progress` doesn't automatically imply that there is a `--progress` flag that can be used. If we are documenting one of them we can mention the other too. Something like `You can use --progress to show query progress` (since that's not default in batch mode AFAIK).